catalogue/webapp/components/Download/Download.js

```diff
@@ -160,9 +160,9 @@ const Download = ({
                   .filter(option => option.format !== 'text/plain') // We're taking out raw text for now
                   .map(option => {
                     const action =
-                      option.label === 'Download full size'
+                      option.label === 'This image (Full size)'
```

Comment:
Just wondering where the language came from? Would download on it's own make sense?

Reply:
You can see the discussion on language used here: #4938